docs/pages-for-subheaders/rke1-hardening-guide.md

```diff
@@ -12,10 +12,11 @@ This hardening guide is intended to be used for RKE clusters and is associated w
 
 | Rancher Version | CIS Benchmark Version | Kubernetes Version           |
 |-----------------|-----------------------|------------------------------|
-| Rancher v2.7    | Benchmark v1.23       | Kubernetes v1.23 up to v1.25 |
+| Rancher v2.7    | Benchmark v1.7       | Kubernetes v1.24 up to v1.25 |
 
 :::note
-At the time of writing, the upstream CIS Kubernetes v1.25 benchmark is not yet available in Rancher. At this time Rancher is using the CIS v1.23 benchmark when scanning Kubernetes v1.25 clusters. Due to that, the CIS checks 5.2.3, 5.2.4, 5.2.5 and 5.2.6 might fail.
+- Since Benchmark v1.24, check id `4.1.7 Ensure that the certificate authorities file permissions are set to 600 or more restrictive (Automated)` might fail, as /etc/kubernetes/ssl/kube-ca.pem is provisioned in 644 by default.
+- Since Benchmark v1.7 (latest), `--protect-kernel-defaults` (check id 4.2.6) parameter is not required anymore, and was replaced.
```

Comment:
```suggestion
- In Benchmark v1.24 and later, check id `4.1.7 Ensure that the certificate authorities file permissions are set to 600 or more restrictive (Automated)` might fail, as `/etc/kubernetes/ssl/kube-ca.pem` is set to 644 by default.
- In Benchmark v1.7, `--protect-kernel-defaults` (check id 4.2.6) parameter isn't required anymore, and was replaced.
```

Similar comment as https://github.com/rancher/rancher-docs/pull/794#discussion_r1302017920, "Should we add what the --protect-kernel-defaults parameter was replaced by? If not, should we remove mention that it was replaced?"

Reply:
@martyav thanks for the review. I think we should remove "and was replaced". In fact, the check id `4.2.6` has been assigned to another check, [here](https://github.com/rancher/security-scan/blob/011ad94bd1a707646e8de880cc1a780f6591b045/package/cfg/rke-cis-1.7-hardened/node.yaml#L283C4-L283C4).